tests/plotting/test_plotting.py

```diff
@@ -3937,11 +3949,12 @@ def setup_plot():
     pl.iren._control_key_release()
     pl.close()
 
-    # shift right click dollys, image 9
+    # shift right click environment rotate, image 9
+    # does nothing here
```

Comment:
I couldn't figure out how to best test this functionality.  In my testing, it only rotates the reflection in an object using `pbr` rendering and using a skybox like in our examples.  I can't get it to also rotate the background like in https://gitlab.kitware.com/vtk/vtk/-/merge_requests/6566.  There is no reproducing code in that MR, so I'm not sure what the difference is.  But I do not think this code is the issue as the reflection does seem to rotate in a manual test.